### DISCUSSION TITLE
[datacoord/sync_segments_scheduler.go:109] ["sync segment with channel failed, retry next ticker"]

### DISCUSSION THREAD

**Discussion by rere950303** [milvus-io/milvus#38018] [Q&A and General discussion]
- milvus version: v2.4.15

The following warning logs are being generated every 5 minutes.

```
[datacoord/sync_segments_scheduler.go:109] ["sync segment with channel failed, retry next ticker"] [collectionID=454190917432377824] [partitionID=454190917432377840] [channel=by-dev-rootcoord-dml_0_454190917432377824v0] [error="context canceled"]
```

**Comment by rere950303** [comment]
@yhmo could you check this bug, please

**Comment by yhmo** [comment]
DataCoord failed to call rpc interface of a data node. The error is "context canceled", which is weird. Is the Milvus cluster or standalone? Is the datacoord restarted every 5 minutes?

**Reply by rere950303** [reply]
@yhmo 
This is running in cluster mode. DataCoord haven't restarted anything and everything is working perfectly fine. However, these warning logs keep appearing. Could this be related to K8s service discovery issues?

**Reply by yhmo** [reply]
It could be a code mistake of a commit on Oct 18th. We are evaluating the impact and will make a fix.

**Reply by rere950303** [reply]
@yhmo thank for your support! 

Could you please mention me when a new release version is available with the fix? Thank you.

**Reply by yhmo** [reply]
Sure. Thank you for catching this defect.

**Reply by rere950303** [reply]
@yhmo Could you share an approximate/tentative release schedule?

**Reply by yhmo** [reply]
The fix has been merged into 2.4 branch, will be released in the next minor version of v2.4
https://github.com/milvus-io/milvus/pull/38032

**Reply by rere950303** [reply]
@yhmo When will the next minor version be released?

**Reply by yhmo** [reply]
Next week, hopefully.
